x-pack/plugins/observability_solution/slo/public/pages/slos/components/slo_list.tsx

```diff
@@ -80,6 +80,7 @@ export function SloList() {
           : ''
       }
       `),
+      starterPrompts: ['What is an SLO?', 'How do I create an SLO?'],
```

Comment:
@CoenWarmer Don't we need to string localization here?

Reply:
Yeah, it's debatable. On the one hand these prompts are meant as an equivalent of what the user might type in, and so from that pov they should be translated. The LLM also understands multiple languages so I'm reasonably sure it will accept the translated value. Ok, I'm convinced :)